### PR TITLE
Add Hindi translations

### PR DESCRIPTION
**Pull request by triach-rold**
**Summary**

_Insert a brief description about what your change is / does._

> If this is related to an issue, please link the issue here

**Translation checklist**

- [Yes] I have used the correct [ISO 639-1 code](https://www.alchemysoftware.com/livedocs/ezscript/Topics/Catalyst/Language.htm) for my translations (e.g. `en`)
- [Yes] I have made sure to leave a `,` at the end of each translation added
- [Yes] I have added the language to `supportedLanguages` in alphabetical order
- [Not yet] I have added the language to the table in README.md in alphabetical order
- [Not applicable yet] (_If you translated the README_) I have translated the README and linked it in the main README.md

### CODE REVIEW

File: auto_py_to_exe/web/js/i18n.js [hunk 1]
```diff
@@ -219,6 +226,7 @@ const translationMap = {
         tr: 'Ayarlar',
         th: 'การตั้งค่า',
         fr: 'Paramètres',
+        hi:'सेटिंग्स',
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:229]: There are a few styling issues in this file. Could you format the file using prettier? [The steps can be found here](https://github.com/brentvollebregt/auto-py-to-exe/blob/master/CONTRIBUTING.md#style-guide).

File: auto_py_to_exe/web/js/i18n.js [hunk 2]
```diff
@@ -2068,6 +2136,10 @@ const supportedLanguages = [
     name: 'Greek (Ελληνικά)',
     code: 'el',
   },
+  {
+    name: 'Hindi(हिंदी)',
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:2140]:
```suggestion
    name: 'Hindi (हिंदी)',
```

File: auto_py_to_exe/web/js/i18n.js [hunk 3]
```diff
@@ -2068,6 +2136,10 @@ const supportedLanguages = [
     name: 'Greek (Ελληνικά)',
     code: 'el',
   },
+  {
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:2139]: Can you add this new language to the README please

**Review by brentvollebregt — CHANGES_REQUESTED** [review]
Looking good, just need to run the formatter, add the language to the README and add a space in the language name 👍

File: auto_py_to_exe/web/js/i18n.js (hunk 3 above)
Comment (triach-rold) [reply auto_py_to_exe/web/js/i18n.js:2139]: Sure. Sorry for the late response.

**Comment by triach-rold** [comment]
Readme has been added, and the space should be commited. I guess it should be done :)

File: README-Hindi.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+| स्पेनी लैटिन (Español Latam)               | [Matyrela](https://github.com/Matyrela)                                                      | UI                                   |
+| थाई (ภาษาไทย)                              | [teerut26](https://github.com/teerut26)                                                      | UI (partial)                         |
+| तुर्की (Türkçe)                            | [mcagriaksoy](https://github.com/mcagriaksoy)                                                | UI and [README](./README-Turkish.md) |
+| यूक्रेनी (Українська)                      | [AndrejGorodnij](https://github.com/AndrejGorodnij)                                          | UI                                   |
+| वियतनामी (Tiếng Việt)                     | [7777Hecker](https://github.com/7777Hecker)                                                  | UI                                   |
+
+> क्या आप किसी अन्य भाषा के लिए अनुवाद जोड़ना चाहते हैं? CONTRIBUTING.md में ["अनुवाद जोड़ें या अद्यतन करें" अनुभाग](./CONTRIBUTING.md#add-or-update-a-translation) का पालन करें।
```
Comment (brentvollebregt) [inline README-Hindi.md:161]: The translation section can be left out of translated readmes

File: README.md
Comment (brentvollebregt) [inline README.md:1]: Can you add a line to the Translations table in here?

File: auto_py_to_exe/web/js/i18n.js (hunk 1 above)
Comment (brentvollebregt) [reply auto_py_to_exe/web/js/i18n.js:229]: This file still need the auto-formatted run on it

File: auto_py_to_exe/web/js/i18n.js (hunk 3 above)
Comment (brentvollebregt) [reply auto_py_to_exe/web/js/i18n.js:2139]: Sorry, I meant a line in the translation table in the README

**Review by brentvollebregt — CHANGES_REQUESTED** [review]

File: README-Hindi.md (hunk 4 above)
Comment (triach-rold) [reply README-Hindi.md:161]: Alright, I'll delete it.

**Comment by brentvollebregt** [comment]
@triach-rold there are still some unresolved comments - just a heads up in case you missed them

**Comment by brentvollebregt** [comment]
`auto_py_to_exe/web/js/i18n.js` is the file that needs to be formatted (the comment is on that file). 
Also [here is an example of the row in the README](https://github.com/brentvollebregt/auto-py-to-exe/blob/master/README.md?plain=1#L134) that you need to add.

File: auto_py_to_exe/web/js/i18n.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
 // Constants used in translation map
-const applicationName = 'auto-py-to-exe';
+const applicationName = "auto-py-to-exe";
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:2]: Can you run `npx prettier --write .` from the root directory so the config in `.prettierrc` is used?

**Review by brentvollebregt — CHANGES_REQUESTED** [review]

**Review by brentvollebregt — APPROVED** [review]

**Comment by brentvollebregt** [comment]
Thanks for the contribution! This has been released in [v2.43.0](https://github.com/brentvollebregt/auto-py-to-exe/releases/tag/v2.43.0) 🚀

**Pull request merged**